python/dasclient/das_agol_tools.py

```diff
@@ -0,0 +1,531 @@
+import logging
+import urllib
+import pytz
+import tempfile
+from datetime import datetime, timedelta
+import dateparser
+from arcgis.gis import GIS
+from arcgis.features import FeatureLayer, Feature
+from arcgis.geometry import Point, Polyline
+from .dasclient import DasClient
+from .version import __version__
+from .schemas import EREvent, ERLocation
+
+class das_agol_tools(object):
```

Comment:
object is assumed. (python 2.6 required it) Here we use camel case when declaring classes, so it would look like, may event shorten it to AgolTools
class DasAgolTools: